README.md

```diff
@@ -125,14 +125,14 @@ Usage: docker volume create [<VOLUME_NAME>]
 <pre>
 $ docker run \
 --detach \
---publish [HOST_PORT]:[CONTAINER_PORT] \
 --volume 9c-volume:/app/data \
-planetariumhq/ninechronicles-headless \
-<a href = "#run" title="NineChronicles Standalone options">[NineChronicles Standalone Options]</a>
+planetariumhq/ninechronicles-headless:latest \
+<a href = "#run" title="NineChronicles Headless options">[NineChronicles Headless Options]</a>
 </pre>
 #### Note)
-* Instead of including the "--ice-server" option, allocate an Elastic IP to your instance and include it as the "--host" option and after adding an inbound port, include it as the "--port" option. Refer to these official docs on [Elastic IP allocation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html) and [editing inbound rules](https://docs.aws.amazon.com/cli/latest/userguide/cli-services-ec2-sg.html) for more information.
-* For mining, make sure to include "--private-key" option with your private key. Also, include "--libplanet-node" to run the default libplanet node. 
+* If you want to use the same headless options as your game client, refer to **`config.json`** under **`%localappdata%\Programs\Nine Chronicles\resources\app`**.
```

Comment:
I'm not sure to comment game client here. And it doesn't comment what the game means.

Reply:
Is it better to say "Nine Chronicles" than "game client"? I assumed that users that would want to run NineChronicles.Headless have already played the game so they would know what "game client" means.